src/config/chain.config.ts

```diff
@@ -15,6 +15,7 @@ import {
   EWC_IDENTITY_MANAGER_ADDRESS,
   EWC_PUBLIC_RESOLVER_ADDRESS,
   EWC_RESOLVER_V1_ADDRESS,
+  VOLTA_RESOLVER_V2_ADDRESS,
```

Comment:
I didn't realize that we weren't using the `V2` Resolver in ICL 🤯 ! @nichonien do you think it would be a good idea if we started using it so that we're better placed to support revocation?

Reply:
@jrhender Yeah, definitely. We need it for revocation.